driver-core/src/test/functional/com/mongodb/client/model/AggregatesFunctionalSpecification.groovy

```diff
@@ -174,6 +174,8 @@ class AggregatesFunctionalSpecification extends OperationFunctionalSpecification
 
         aggregate([group('$z')]).containsAll([new Document('_id', true), new Document('_id', false)])
 
+        aggregate([group(null, Accumulators.count('acc'))]) == [new Document('_id', null).append('acc', 3)]
```

Comment:
Note that because there is already a `Aggregates.count(String)` method, we (and our users) can't just say `count('acc`)` here like we can with the other accumulators.  This naming conflict is unfortunate.